cmd/archive_upload_server/server.go

```diff
@@ -106,10 +107,11 @@ func (r rvServer) FileUpload(ctx context.Context, req *pb.FileRequest) (*pb.File
 	// Process the content based upon project requirements.
 	switch {
 	case proj == pb.FileRequest_ROUTEVIEWS:
+		return r.handleDataFile(ctx, resp, fn, content)
 	case proj == pb.FileRequest_RIPE_RIS:
 	case proj == pb.FileRequest_RPKI_RARC:
 		// Simply store the file.
-		return r.handleRPKIRarc(ctx, resp, fn, content)
+		return r.handleDataFile(ctx, resp, fn, content)
```

Comment:
It looks like we are writing to the same bucket monitored by the converter service. Since the RPKI files are JSON VRPs, they will not be converted (JSON, not bz2) and stored in BigQuery for the moment. Are these just snapshots of VRPs (so we can correlate the routes with RPKI statuses in the past)...? What's your plan about this?

Reply:
I would like to eventually be able to ingest the VRPs into BQ as well, that's not super important today.
I think 'today' the important thing is to at least have an archive we can ingest into BQ later:) or just download and grep (in the worst case).